01-html/03-caitlinandcollin.html

```diff
@@ -2,4 +2,228 @@
+          playing music on boats, shredding powder on mountains, or drinking scotch in the sunshine. Currently located in Vancouver, BC.</p>
+      </article>
+  </section>
+  <section class="inline-block">
+    <article>
+      <p>
+        <span class="name">JONATHAN FLEMMING</span>
+        <img src="http://static.comicvine.com/uploads/original/11124/111240586/4969419-kylo+ren+afbeelding+2.jpg" alt="kylo" width="300">
+      </p>
+      <p>Jonathan and I met in college as restaurant workers, and were fast friends by the end of our time at UW. We've been to more major sporting events than I can conveniently count, played more hours of poker than I care to admit, and fostered our mutual
+        love of scotch. Jonathan is currently the general manager at the fantastic Bar Sajor in Pioneer Square, he is marrying his love, Libby, next year.</p>
+    </article>
+    <article>
+      <p>
+        <img src="http://static.comicvine.com/uploads/original/11124/111240586/4969419-kylo+ren+afbeelding+2.jpg" alt="kylo" width="300">
+        <span class="name">RYLAN BAUERMEISTER</span>
+      </p>
+      <p>Rylan, in addition to being my future brother-in-law, is a hell of a writer, one of the few people who understands my love of video games, and not the person you want to chase up a hiking trail. When he's not penning the next great American novel,
+        Rylan is volunteering at a wildlife animal shelter.</p>
+    </article>
+    <article>
+      <p>
+        <span class="name">BRETT WISE</span>
+        <img src="http://static.comicvine.com/uploads/original/11124/111240586/4969419-kylo+ren+afbeelding+2.jpg" alt="kylo" width="300">
+      </p>
+      <p>Brett is one of my oldest friends. We've had some truly good times together, from the (in)famous trip to the Rose Parade in Pasadena and an epic high school senior trip to Hawaii, to Monday Night Raw at the Tacoma Dome and summers on the lake. Brett
+        currently works in sales at Coastal Wear Products, and had two freakishly cute children with his wife, Bethany.</p>
+    </article>
+    <article>
+      <p>
+        <img src="http://static.comicvine.com/uploads/original/11124/111240586/4969419-kylo+ren+afbeelding+2.jpg" alt="kylo" width="300">
+        <span class="name">DAN KENNY</span>
+      </p>
+      <p>Dan is the man who is truly down to do anything, at anytime - whether it's a late night drink or a road trip, Dan will make it happen. He is a great golfer, a drinker of delicious scotch, and, as I found out last summer, one hell of a kickball player.
+        Dan is an attorney at Christie Law Group.</p>
+    </article>
+    <article>
+      <p>
+        <span class="name">GARRETT AMINI</span>
+        <img src="http://static.comicvine.com/uploads/original/11124/111240586/4969419-kylo+ren+afbeelding+2.jpg" alt="kylo" width="300">
```

Comment:
Is it super cocky of me to make you code up a page with my face on it? 
...
also, do I kinda look like Kylo Ren? Maybe if I broke my nose.